src/routes/addUserAsVIP.ts

```diff
@@ -38,12 +39,12 @@ export async function addUserAsVIP(req: AddUserAsVIPRequest, res: Response): Pro
     try {
         if (enabled && !userIsVIP) {
             // add them to the vip list
-            await db.prepare("run", 'INSERT INTO "vipUsers" VALUES(?)', [userID]);
+            VipRepository.addVip(userID, db);
```

Comment:
What may make more sense is choosing which db to use (`db` or `privateDB`) based on the query. There is no `vipUsers` table in the public db.